web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js

```diff
@@ -0,0 +1,39 @@
+import { addConsolidatedProperties } from './addConsolidatedProperties';
+
+describe('addConsolidatedProperties', () => {
+  it('should return the expected object when the message passed in is on the lead case & consolidated group', () => {
```

Comment:
nit-pick: Please change the test description to account for the new properties being added. For example `should add the Lead Case tooltip text and set both inConsolidatedGroup and isLeadCase  to true when docket number and leadDocketNumber are the same`

Reply:
changed